daemon/oci_linux.go

```diff
@@ -900,6 +900,16 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	s.Process.NoNewPrivileges = c.NoNewPrivileges
 	s.Process.OOMScoreAdj = &c.HostConfig.OomScoreAdj
 	s.Linux.MountLabel = c.MountLabel
+	if s.Annotations == nil {
+		s.Annotations = make(map[string]string)
+	}
+	for k, v := range c.Config.Labels {
+		if oldValue, ok := s.Annotations[k]; ok {
+			logrus.Errorf("Key %q already exists in Annotations, the new value will be ignored, old value=%q, new value=%q", k, oldValue, v)
+			continue
```

Comment:
oh; meant this to be returning the error (thus "fail");

```go
return nil, errors.Errorf(".......
```

Reply:
@thaJeztah  I thought that this should not terminate the container starting, just give an error message, since the `Annotations` does not seem to be used directly by docker.
But returning the error is also reasonable here. Have done. :-)